astrology.lic

```diff
@@ -121,10 +124,29 @@ class Astrology
     waitrt?
   end
 
+  def roll_bones
+    if @divination_bones_tied
+      bput('untie bones from my '+@divination_bones_container, 'divination bones')
```

Comment:
When possible, we prefer string interpolation (something like `"untie bones from my #{@divination_bones_container}"`).